crypto/src/lib.rs

```diff
@@ -574,18 +577,15 @@ impl<'de> Deserialize<'de> for PrivateKey {
     }
 }
 
-#[cfg(feature = "ffi")]
+#[cfg(any(feature = "ffi_api", feature = "ffi"))]
 mod ffi {
-    use iroha_ffi::{gen_ffi_impl, handles};
-
-    use super::{KeyPair, PrivateKey, PublicKey};
-
-    handles! {0, KeyPair, PublicKey, PrivateKey}
+    use super::*;
 
-    gen_ffi_impl! { Clone: KeyPair, PublicKey, PrivateKey}
-    gen_ffi_impl! { Eq: KeyPair, PublicKey, PrivateKey}
-    gen_ffi_impl! { Ord: PublicKey }
-    gen_ffi_impl! { Drop: KeyPair, PublicKey, PrivateKey}
+    iroha_ffi::handles! {0, KeyPair, PublicKey, PrivateKey}
```

Comment:
This syntax is still confusing me... What is `0` here? Maybe rename the macro or update the syntax?

Reply:
I've updated the description